include/ur_client_library/comm/pipeline.h

```diff
@@ -319,7 +319,7 @@ class Pipeline
   }
 
   /*!
-   * \brief Returns the next package in the queue. Can be used instead of registering a consumer.
+   * \brief Returns the most recent package in the queue. Can be used instead of registering a consumer.
```

Comment:
```suggestion
   * \brief Returns the most recent package in the queue. Can be used instead of registering a consumer. If the queue already contains one or more items, the queue will be flushed and the newest item will be returned. If there is no item inside the queue, the function will wait for \p timeout for a new package
```

Reply:
Yes, I would agree with that.